karma.conf.js

```diff
@@ -15,143 +15,101 @@ See the License for the specific language governing permissions and
 
 const SL_LAUNCHERS = {
-  'sl-chrome-stable': {
+  'sl-ie': {
     base: 'SauceLabs',
-    browserName: 'chrome',
-    version: 'latest',
-    platform: 'macOS 10.12',
+    browserName: 'internet explorer',
+    version: '11',
+    platform: 'Windows 8.1',
   },
-  'sl-chrome-beta': {
+  'sl-edge-17': {
     base: 'SauceLabs',
-    browserName: 'chrome',
-    version: 'dev',
-    platform: 'macOS 10.12',
+    browserName: 'microsoftedge',
+    version: '17',
+    platform: 'Windows 10',
   },
-  'sl-chrome-previous': {
+  'sl-edge-15': {
     base: 'SauceLabs',
-    browserName: 'chrome',
-    version: 'latest-1',
-    platform: 'macOS 10.12',
+    browserName: 'microsoftedge',
+    version: '15',
+    platform: 'Windows 10',
   },
-  'sl-firefox-stable': {
+  'sl-safari-11': {
     base: 'SauceLabs',
-    browserName: 'firefox',
-    version: 'latest',
-    platform: 'Windows 10',
+    browserName: 'safari',
+    version: '11',
+    platform: 'macOS 10.13',
   },
-  'sl-firefox-previous': {
+  'sl-safari-10': {
     base: 'SauceLabs',
-    browserName: 'firefox',
-    version: 'latest-1',
-    platform: 'Windows 10',
+    browserName: 'safari',
+    version: '10',
+    platform: 'OS X 10.12',
   },
-  'sl-ie': {
+  'sl-safari-9': {
     base: 'SauceLabs',
-    browserName: 'internet explorer',
-    version: '11',
-    platform: 'Windows 8.1',
+    browserName: 'safari',
+    version: '9',
+    platform: 'OS X 10.11',
   },
-  // TODO(sgomes): Re-enable Edge and Safari after Sauce Labs problems are fixed.
-  // 'sl-edge': {
-  //   base: 'SauceLabs',
-  //   browserName: 'microsoftedge',
-  //   version: 'latest',
-  //   platform: 'Windows 10',
-  // },
-  // 'sl-safari-stable': {
-  //   base: 'SauceLabs',
-  //   browserName: 'safari',
-  //   version: 'latest',
-  //   platform: 'macOS 10.12',
-  // },
-  // 'sl-safari-previous': {
-  //   base: 'SauceLabs',
-  //   browserName: 'safari',
-  //   version: '9.0',
-  //   platform: 'OS X 10.11',
-  // },
-  'sl-ios-safari-latest': {
+  'sl-chrome-41': {
     base: 'SauceLabs',
-    deviceName: 'iPhone Simulator',
-    platformVersion: '10.0',
-    platformName: 'iOS',
-    browserName: 'Safari',
+    browserName: 'chrome',
+    version: '41',
+    platform: 'Linux'
   },
-  // 'sl-ios-safari-previous': {
-  //   base: 'SauceLabs',
-  //   deviceName: 'iPhone Simulator',
-  //   platformVersion: '9.3',
-  //   platformName: 'iOS',
-  //   browserName: 'Safari',
-  // },
 };
 
 module.exports = function(config) {
   config.set({
     basePath: '',
-    frameworks: ['mocha'],
+    frameworks: ['mocha', 'chai'],
     files: [
       {pattern: 'node_modules/@webcomponents/webcomponentsjs/custom-elements-es5-adapter.js', watched: false},
-      'test/unit/index.js',
+      {pattern: 'node_modules/@webcomponents/webcomponentsjs/webcomponents-bundle.js', watched: false},
+      {pattern: 'test/unit/mwc-{button,fab,radio,ripple}*.js', watched: false},
```

Comment:
Why only disable file watching on `button,fab,radio,ripple`?

Reply:
This is actually only _testing` button, fab, radio, and ripple